fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -69,6 +73,7 @@ def __init__(
         process_group: Any = None,
         broadcast_buffers: bool = True,
         sync_models_at_startup: bool = True,
+        reduce_buffer_size: int = 2 ** 23,
```

Comment:
n00b question: is there a good rule of thumb for how to tune this parameter?

Reply:
not that I know really, it would depend on the speed/latency of the interconnect in between the nodes, and the amount of memory that you're ready to loose (this kills the gradient sharding basically)

Reply:
I tested this value and it provides a very significant speedup vs. previous status, but I did not have time to do a sweep honestly